Quorum/checks/diff.py

```diff
@@ -123,21 +123,29 @@ def __print_diffs_results(self, missing_files: list[SourceCode], files_with_diff
             missing_files (list[SourceCode]): A list of SourceCode objects representing missing files.
             files_with_diffs (list[Compared]): A list of Compared objects representing files with differences.
         """
-        total_number_of_files = len(self.source_codes)
-        number_of_missing_files = len(missing_files)
-        number_of_files_with_diffs = len(files_with_diffs)
-
-        msg = f"Compared {total_number_of_files - number_of_missing_files}/{total_number_of_files} files for proposal {self.proposal_address}"
-        if number_of_missing_files == 0:
-            pp.pretty_print(msg, pp.Colors.SUCCESS)
-        else:
-            pp.pretty_print(msg, pp.Colors.WARNING)
-            for source_code in missing_files:
-                pp.pretty_print(f"Missing file: {source_code.file_name} in local repo", pp.Colors.WARNING)
-
-        if number_of_files_with_diffs == 0:
-            pp.pretty_print("No differences found.", pp.Colors.SUCCESS)
-        else:
-            pp.pretty_print(f"Found differences in {number_of_files_with_diffs} files", pp.Colors.FAILURE)
-            for compared_pair in files_with_diffs:
-                pp.pretty_print(f"Local: {compared_pair.local_file}\nProposal: {compared_pair.proposal_file}\nDiff: {compared_pair.diff}", pp.Colors.FAILURE)
+        num_total_files = len(self.source_codes)
+        num_missing_files = len(missing_files)
+        num_diff_files = len(files_with_diffs)
+        num_identical = num_total_files - num_missing_files - num_diff_files
+
+        # Identical files message.
+        pp.pprint(f'Files found identical: {num_identical}/{num_total_files}', pp.Colors.SUCCESS)
+
+        # Diffs files message.
+        if num_diff_files > 0:
+            diffs_msg = ('Proposal files found to deviate from their source of truth counterpart: '
+                        f'{num_diff_files}/{num_total_files}\n')
+            for i, compared_pair in enumerate(files_with_diffs, 1):
+                diffs_msg += (f'{i}. Proposal file: {compared_pair.proposal_file}\n'
+                              f'Source of truth file: {compared_pair.local_file}\n'
+                              f'Diff can be found here: {compared_pair.diff}\n')
```

Comment:
```suggestion
                diffs_msg += (f'{i}.\tProposal file: {compared_pair.proposal_file}\n'
                              f'\tSource of truth file: {compared_pair.local_file}\n'
                              f'\tDiff can be found here: {compared_pair.diff}\n')
```
This will look nicer

Reply:
Yeah, I wasn't too sure about the tabs. Last time I tried printing tabs it looked bad for some reason. I'll try that again in the entire formatting and update if it will actually look good.